api/flask_api.py

```diff
@@ -3,24 +3,30 @@
 """
 import os
 from flask import Flask
-from flask_restful import Api
+from flask_restful import Api, Resource
 from mongoengine import connect
 # You need to implement two resources: Brevet and Brevets.
 # Uncomment when done:
-# from resources.brevet import Brevet
-# from resources.brevets import Brevets
+from resources.brevet import BrevetName
+from resources.brevets import BrevetsName
+
 
 # Connect MongoEngine to mongodb
 connect(host=f"mongodb://{os.environ['MONGODB_HOSTNAME']}:27017/brevetsdb")
 
 # Start Flask app and Api here:
-# app = 
-# api = 
+app = Flask(__name__)
+api = Api(app)
 
```

Comment:
```suggestion
port_num = True if "PORT" not in os.environ else os.environ["PORT"]
```
Allows us to test now if the PORT environment variable exists rather then just assuming.